script/update

```diff
@@ -252,38 +248,16 @@ def setup_mips64el_toolchain(target_arch):
   subprocess.check_call(['chromium/scripts/copy_config.sh'], cwd=ffmpeg_dir)
   subprocess.check_call(['chromium/scripts/generate_gn.py'], cwd=ffmpeg_dir)
 
-def run_gn(target_arch):
-  env = os.environ.copy()
-  if sys.platform in ['win32', 'cygwin']:
-    env['DEPOT_TOOLS_WIN_TOOLCHAIN'] = '0'
-
-  target_os = {
-    "cygwin": "win",
-    "darwin": "mac",
-    "linux2": "linux",
-    "win32": "win",
-  }.get(sys.platform)
 
-  if target_arch == 'ia32':
-    target_cpu = 'x86'
-  else:
-    target_cpu = target_arch
+def run_gn(target_arch):
+  gn_args = {}
```

Comment:
@MarshallOfSound 
After this change adding `cc_wrapper="sccache"` should be as simple as:

```python
if use_sccache:
  gn_args['cc_wrapper'] = ''sccache"
```